src/System.Collections/src/System.Collections.builds

```diff
@@ -8,6 +8,9 @@
     <Project Include="System.Collections.csproj" Condition="'$(TargetsWindows)' == 'true'">
       <AdditionalProperties>TargetGroup=net46</AdditionalProperties>
     </Project>
+    <Project Include="System.Collections.csproj" Condition="'$(TargetsWindows)' == 'true'">
+      <AdditionalProperties>TargetGroup=netcore50aot</AdditionalProperties>
```

Comment:
If this is building against contracts please use netcore50 as the target group because it will be the same for both netcore50 and netcore50aot. We should only ever use netcore50aot for things that need to reference things from the Private.NetNative targeting pack.